java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -4526,9 +4586,9 @@ public void bitop(BaseClient client) {
         // First bit is flipped to 1 and throws 'utf-8' codec can't decode byte 0x9e in position 0:
         // invalid start byte
         // TODO: update once fix is implemented for https://github.com/aws/glide-for-redis/issues/1447
-        // ExecutionException executionException =
-        //        assertThrows(ExecutionException.class, () -> client.get(destination).get());
-        // assertTrue(executionException.getCause() instanceof RuntimeException);
+        ExecutionException executionException =
+                assertThrows(ExecutionException.class, () -> client.get(destination).get());
+        assertTrue(executionException.getCause() instanceof RuntimeException);
```

Comment:
I think you need to replace that with `client.get(gs(destination))` which should succeed
And remove the comment above in that case